zabdig

```diff
@@ -69,6 +69,11 @@ from fnmatch import fnmatch
 from warnings import warn
 
 
+CONF_PATH = f'{os.environ.get('XDG_CONFIG_HOME', '~/.config')}/.zabbixrc'
+if not os.path.exists(os.path.expanduser(CONF_PATH)):
+    CONF_PATH = ('~/.zabbixrc')
```

Comment:
Remove the extra parentheses.

Reply:
Which extra parentheses are you referring to?

Reply:
Those on line 74